### ISSUE TITLE
Add celebratory image to "Site Creation" checklist step 

### ISSUE THREAD

**Issue by getdave** [Automattic/wp-calypso#43938]
> Would it make sense to add an image to this step to bump up the celebratory nature little bit up?

@simison Quite possibly. Seems like a good suggestion. Currently, I'm working to [this design](https://github.com/Automattic/wp-calypso/pull/43861#issuecomment-653317424) by @danhauk. If we want/need to add this then it should probably be in a followup PR.

If @danhauk agrees then I'm happy to raise a separate Issue.

_Originally posted by @getdave in https://github.com/Automattic/wp-calypso/pull/43861#issuecomment-654730148_

**Comment by jancavan** [comment]
Yes, I agree that this one card could be a little bit more celebratory. We can add the illustration back like so:

![Screen Shot 2020-07-07 at 11 32 30 AM](https://user-images.githubusercontent.com/4924246/86826414-8f704f00-c045-11ea-906f-a04f443a6dcb.png)

I feel like the dark gray pill is a little too close to the heading. I think we should move it down a bit? @getdave @danhauk

**Comment by danhauk** [comment]
I agree. When this is picked up, could we make the task component have an optional image prop. I'm thinking of an additional use case would be as a video placeholder to open a support doc in a modal (something I was exploring previously as a future enhancement):

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/448298/86840799-24d50880-c071-11ea-83bb-a9c85125ce59.png">

**Comment by jancavan** [comment]
@getdave I updated the [mockup above](https://github.com/Automattic/wp-calypso/issues/43938#issuecomment-655048212). I think the illustration can be made slightly bigger. Thanks.
